luigi_pipeline/lib/hail_tasks.py

```diff
@@ -84,12 +96,52 @@ def import_vcf(self):
             recode = {f"{i}": f"chr{i}" for i in (list(range(1, 23)) + ['X', 'Y'])}
         elif self.genome_version == "37":
             recode = {f"chr{i}": f"{i}" for i in (list(range(1, 23)) + ['X', 'Y'])}
+        mt = hl.import_vcf(
+            self.source_paths,
+            reference_genome=self.genome_version,
+            skip_invalid_loci=True,
+            contig_recoding=recode,
+            force_bgz=True,
+            min_partitions=500,
+        )
+        if self.wes_filter_source_paths:
+            filters_ht = hl.import_vcf(
+                self.wes_filter_source_paths,
+                reference_genome=self.genome_version,
```

Comment:
Looks like this line changed from the below: (`reference_genome='GRCh' + self.genome_version` -> `reference_genome=self.genome_version`).

Should we be using `hail_utils.py`'s version of `import_vcf` here anyways?

Reply:
(I meant to comment this a few lines up, but it's the same in both places).